netci.groovy

```diff
@@ -0,0 +1,46 @@
+// Import the utility functionality.
+import jobs.generation.Utilities;
+
+def project = 'microsoft/dotnet-apiport'
+def branch = 'master'
```

Comment:
These two lines should pull from incoming parameters:

def project = GithubProject
def branch = GithubBranchName